ibis/expr/window.py

```diff
@@ -228,10 +274,43 @@ def trailing_window(periods, group_by=None, order_by=None):
     -------
     win : ibis Window
     """
-    return Window(preceding=periods, following=0,
+    return Window(preceding=rows, following=0,
                   group_by=group_by, order_by=order_by)
 
 
+def trailing_time_window(prior_interval, order_by, group_by=None):
```

Comment:
I think what we'll want to do here is make this a call into `trailing_window` so that backends can handle the casting and unit conversion logic individually. That means that `trailing_window` will need to take a `how` argument.